### PR TITLE
ci: support @typescript-eslint/no-inferrable-types

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [x] Build or CI related changes
- [ ] Documentation content changes

## What is the new behavior?

image

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ module.exports = {
     rules: {
         // TODO: move rules to @tinkoff/eslint-config-angular
         '@typescript-eslint/no-useless-constructor': 'off',
+        '@typescript-eslint/no-inferrable-types': ['error', {ignoreParameters: true}],
```
Comment (splincode) [inline .eslintrc.js:14]: not sure about `ignoreParameters`

Comment (splincode) [reply .eslintrc.js:14]: @waterplea what do you think? do we need it?

`ignoreParameters = false`

image

image

image

Comment (waterplea) [reply .eslintrc.js:14]: Let's keep type for parameters so it's clear what function expects at a glance.

**Review by waterplea — APPROVED** [review]

**Comment by splincode** [comment]
@vladimirpotekhin please, review me)

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**